go.mod

```diff
@@ -1,76 +1,15 @@
 module github.com/mayadata-io/openebs-operator
```

Comment:
Next PR: rename the module to mayadata.io/oep-operator

Reply:
Done
Used `mayadata.io/openebs-upgrade`.